### DISCUSSION TITLE
Material Limit - Transient Infiltration

### DISCUSSION THREAD

**Discussion by ZeidBit** [Anura3D/Anura3D_OpenSource#15] [General]
Hey All, 

I was wondering if there is some kind of material limit when analyzing slope failure under infiltration. It seems with 4 materials it works fine but with more than 4 the liquid pressure and stresses don't make sense anymore and there is no displacement after infiltration (heavy rainfall of -2e-4).

**Comment by murphy-john** [comment]
Hello @ZeidBit, I asked this question at our developers meeting. There was no known limitation to the number of materials to use transient infiltration. The advise was that in models with that many materials it become very complex and to evaluate the need for the additional materials. One member of the group mentioned there is sometimes an issue where different "material types" are used in the same model. 

Other than that, you'll have to try to do some debugging in Visual Studio to try to find the issue.

**Comment by n4nahm** [comment]
Hi guys, I am following up on this discussion:
I am using Anura3D 2022 and MPM to model the runout of an embankment. The embankment is saturated with water below a certain depth. So, I have a soil surface profile, phreatic surface, and also free water standing near the toe in a canal (See attached image).

I am following tutorial 11 from the manual to define the "Soil surface". Does the "soil surface" go above or below the free water ?

![image](https://user-images.githubusercontent.com/108018180/175097841-ed5d7cbb-9da2-4179-8d21-15ea57d58445.png)

**Reply by ayerrocolom** [reply]
Hello,

The K0 stress initialization is only available for one homogeneous material (read limitations in the [Tutorial manual](https://github.com/Anura3D/Anura3D_OpenSource/blob/main/manuals/TutorialManual_2022.pdf), page 6), so the initialization of phreatic surface and soil surface will not work properly in this scenario. Also, the combined use of unsaturated materials with water has not been validated yet.
I am afraid that the problem that you are attempting here is beyond the current capabilities of Anura3D and further implementations and validations are needed. If you are willing to further develop the code, you can keep us posted!

Best,
Alba

**Reply by ZeidBit** [reply]
Hey, 

Is there a way to incorporate water into the slope stability problem then? I know we can define a very stiff base and specify contact properties but I would like to have realistic materials instead.  I get the error where the particle lies outside the mesh whenever I attempt to have water slide over a non-stiff material. 

Thanks, 
Zeid
